.prettierrc

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
Made things easier when working in the demos with Prettier configured (which is now default in VSCode) without going all the way to configuring linting for each of them. Happy to remove is this is problematic though.